app/controllers/users_controller.rb

```diff
@@ -2,9 +2,33 @@ class UsersController < ApplicationController
   before_action :set_no_cache_header
   before_action :raise_suspended, only: %i[update]
   before_action :set_user, only: %i[update update_twitch_username update_language_settings confirm_destroy request_destroy full_delete remove_association]
-  after_action :verify_authorized, except: %i[signout_confirm add_org_admin remove_org_admin remove_from_org]
+  after_action :verify_authorized, except: %i[index signout_confirm add_org_admin remove_org_admin remove_from_org]
   before_action :authenticate_user!, only: %i[onboarding_update onboarding_checkbox_update]
 
+  DEFAULT_FOLLOW_SUGGESTIONS = %w[ben jess peter maestromac andy liana].freeze
+
+  def index
+    if !user_signed_in? || less_than_one_day_old?(current_user)
```

Comment:
@rhymes As mentioned on Slack, I think the `user_signed_in?` check should stay, as we use it to display defaults and then early return. Or am I missing something?